packages/design-toolkit/src/components/flashcard/index.tsx

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use client';
+import { clsx } from '@accelint/design-foundation/lib/utils';
+import 'client-only';
+import { createContext, useContext } from 'react';
+import { DetailsList } from '../details-list';
+import { DetailsListLabel } from '../details-list/label';
+import { DetailsListValue } from '../details-list/value';
+import { Skeleton } from '../skeleton';
+import styles from './styles.module.css';
+import type {
+  FlashcardComponentProps,
+  FlashcardDetailsListProps,
+  FlashcardProps,
+} from './types';
+
+export const FlashcardContext = createContext<FlashcardProps>({
+  isLoading: false,
+});
+
+/**
+ * Example usage.
+ *
+ * ```tsx
+ * <Flashcard isLoading={isLoading}>
+ *  <FlashcardHero>
+ *    <FlashcardHeader>
+ *      {header}
+ *    </FlashcardHeader>
+ *    <FlashcardSubheader>
+ *      {subHeader}
+ *    </FlashcardSubheader>
+ *  </FlashcardHero>
+ *  <FlashcardAdditionalData>
+ *    {secondaryData}
+ *  </FlashcardAdditionalData>
+ *  <FlashcardDetailsList>
+ *    {detail.map((detail) => (
+ *     <Fragment id={detail.id}>
+ *       <FlashcardDetailLabel>
+ *         {detail.label}
+ *       </FlashcardDetailsLabel>
+ *       <FlashcardDetailsValue>
+ *         {detail.value}
+ *       </FlashcardDetailsValue>
+ *     </Fragment>
+ *     ))}
+ *  </FlashcardDetailsList>
+ * </Flashcard>
+ * ```
+ */
+export function Flashcard(props: FlashcardProps) {
+  const { isLoading, children, className, ...rest } = props;
+
+  return (
+    <FlashcardContext.Provider value={{ isLoading }}>
+      <div {...rest} className={clsx(styles.container, className)}>
+        {children}
+      </div>
+    </FlashcardContext.Provider>
+  );
+}
+Flashcard.displayName = 'Flashcard';
+
+export function FlashcardHero(props: FlashcardComponentProps) {
+  const { children, className, ...rest } = props;
+  const { isLoading } = useContext(FlashcardContext);
+
+  if (isLoading) {
+    return (
+      <div {...rest} className={clsx(styles.hero, className, 'gap-s')}>
+        <Skeleton
+          className={clsx(styles.skeleton)}
```

Comment:
no need for clsx when only one class present

```suggestion
          className={styles.skeleton}
```